Jenkinsfile

```diff
@@ -3,16 +3,40 @@
 node {
     checkout scm
 
-    nodeEnv = docker.image("node:10")
+    nodeEnv = docker.image("node:10-stretch")
     workspace = pwd()
+
+    // Tag used when deploying to NPM.
+    npmTag = "latest"
+
+    // Git branch which releases are deployed from.
+    releaseFromBranch = "master"
+
+    // Pre-release suffix added to new package version number when deploying.
+    // If this is empty, the new deployed version will become the live version.
+    //
+    // Note that once an npm package has been published with a given version,
+    // it is *not* possible to overwrite that version in future (eg. you cannot
+    // publish "v1.1-testing" twice).
+    versionSuffix = ""
+
+    if (versionSuffix != "") {
+        npmTag = "prerelease"
+    }
+
+    pkgName = sh (
+      script: 'cat package.json | jq -r .name',
+      returnStdout: true
+    ).trim()
+
     pkgVersion = sh (
       script: 'cat package.json | jq -r .version',
       returnStdout: true
     ).trim()
 
     stage('Build') {
         nodeEnv.inside("-e HOME=${workspace}") {
-            sh "echo Building Hypothesis client \"${pkgVersion}\""
+            sh "echo Building Hypothesis client"
```

Comment:
Is there a reason this line uses double quotes and the following are single? Interpolation of something?